### PR TITLE
API updates

### PR DESCRIPTION
**Pull request by armandojaleo**
~~Add new properties for payment.processing response~~

~~[Add properties for payment.processing response by yelena-pakumirzakova-cko · Pull Request #820 · checkout/checkout-api-reference](https://github.com/checkout/checkout-api-reference/pull/820)~~

### Add SEPA source APM for NAS

[Add SEPAV4 sources by andre-lopes-cko · Pull Request #829 · checkout/checkout-api-reference](https://github.com/checkout/checkout-api-reference/pull/829)

### CODE REVIEW

File: lib/Checkout/Payments/Request/Source/Apm/RequestSepaSource.php [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+class RequestSepaSource extends AbstractRequestSource
+{
+    public function __construct()
+    {
+        parent::__construct(PaymentSourceType::$sepa);
+    }
+
+    /**
+     * @var Country
```
Comment (Armando-Ibarra) [inline lib/Checkout/Payments/Request/Source/Apm/RequestSepaSource.php:18]: Please check the SDK it should be `* @var string values of Country`

File: lib/Checkout/Payments/Request/Source/Apm/RequestSepaSource.php [hunk 2]
```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Checkout\Payments\Request\Source\Apm;
+
+use Checkout\Common\AccountHolder;
+use Checkout\Common\Address;
```
Comment (Armando-Ibarra) [inline lib/Checkout/Payments/Request/Source/Apm/RequestSepaSource.php:6]: unused import

**Review by Armando-Ibarra — CHANGES_REQUESTED** [review]

File: lib/Checkout/Payments/Request/Source/Apm/RequestSepaSource.php (hunk 2 above)
Comment (armandojaleo) [reply lib/Checkout/Payments/Request/Source/Apm/RequestSepaSource.php]: Removed

File: lib/Checkout/Payments/Request/Source/Apm/RequestSepaSource.php (hunk 1 above)
Comment (armandojaleo) [reply lib/Checkout/Payments/Request/Source/Apm/RequestSepaSource.php]: Ok 👍

**Review by Armando-Ibarra — APPROVED** [review]
Good Job!

**Pull request merged**